src/client/services/formationInitiale/formationInitiale.service.ts

```diff
@@ -1,17 +1,33 @@
+import { ParsedUrlQuery, stringify } from 'querystring';
+
 import { FormationInitialeQueryParams } from '~/client/hooks/useFormationInitialeQuery';
 import { HttpClientService } from '~/client/services/httpClient.service';
 import { Either } from '~/server/errors/either';
 import { FormationInitiale } from '~/server/formations-initiales/domain/formationInitiale';
+import { removeUndefinedKeys } from '~/server/removeUndefinedKeys.utils';
 
 export interface FormationInitialeInterface {
 	rechercherFormationInitiale(query: FormationInitialeQueryParams): Promise<Either<Array<FormationInitiale>>>
 }
 
+interface FormationInitialeQueryFiltre extends ParsedUrlQuery {
```

Comment:
```suggestion
interface FormationInitialeUrlQuery extends ParsedUrlQuery {
```

pour mieux faire la distinction entre l'objet query et la string qu'il y a dans l'url